specification/ai/OpenAI.Assistants/tools/models.tsp

```diff
@@ -82,7 +82,17 @@ model RequiredFunctionToolCall extends RequiredToolCall {
   type: "function";
 
   @doc("Detailed information about the function to be executed by the tool that includes name and arguments.")
-  function: FunctionDefinition;
+  function: RequiredFunctionToolCallDetails;
+}
+
+@added(ServiceApiVersions.v2024_02_15_preview)
+@doc("The detailed information for a function invocation, as provided by a required action invoking a function tool, that includes the name of and arguments to the function.")
+model RequiredFunctionToolCallDetails {
+  @doc("The name of the function.")
+  name: string;
+
+  @doc("The arguments to use when invoking the named function, as provided by the model.")
```

Comment:
Maybe here we could quickly mention that this will be a string containing JSON. But maybe that's implied.

Reply:
Certainly doesn't seem like a bad thing to include! Great suggestion.